.github/workflows/scheduled-vuln-check.yaml

```diff
@@ -3,7 +3,7 @@ name: Scheduled Vulnerability Check
 on:
   schedule:
     # UTC
-    - cron: '0 19 * * WED,SAT'
+    - cron: '0 18 * * WED,SAT'
```

Comment:
Just curious. Any reason for the change?

Reply:
@feeblefakie Thanks for the question. I just wanted to make it slightly different from the check time on the `scalar` side.